src/util/DbrHelper.sol

```diff
@@ -77,12 +78,21 @@ contract DbrHelper is Ownable, ReentrancyGuard {
         uint invAmount
     );
     event MarketApproved(address indexed market);
+    event NewPendingGov(address indexed oldPendingGov, address indexed newPendingGov);
+    event NewGov(address indexed oldGov, address indexed newGov);
+    event NewCurvePool(address indexed newPool, uint256 dolaIndex, uint256 dbrIndex, uint256 invIndex);
 
-    constructor() Ownable(msg.sender) {
-        DBR.approve(address(CURVE_POOL), type(uint).max);
+    constructor(address _curvePool) Ownable(msg.sender) {
+        curvePool = ICurvePool(_curvePool);
+        DBR.approve(address(curvePool), type(uint).max);
         INV.approve(address(INV_MARKET), type(uint).max);
     }
```

Comment:
Coninued to avoid human/gov error:
```suggestion
    constructor(address _curvePool) Ownable(msg.sender) {
        curvePool = ICurvePool(_curvePool);
        for(uint i; i < 3; i++){
            if(curvePool.coins[i] == address(DOLA)){
                dolaIndex = i;
            }
            else if(curvePool.coins[i] == address(DBR)){
                dbrIndex = i;
            }
            else if(curvePool.coins[i] == address(INV)){
                invIndex = i;
            }
        }
        require(dolaIndex + dbrIndex + invIndex == 6, "Incorrect indices");
        DBR.approve(address(curvePool), type(uint).max);
        INV.approve(address(INV_MARKET), type(uint).max);
    }
```